artemis/modules/data/lfi_detector_payloads.txt

```diff
@@ -0,0 +1,861 @@
+/etc/passwd
```

Comment:
Are they taken from somewhere? If yes, can you provide the source and its license as a comment in this file? (and lfi_detector_data can strip comments)